recipes/datarobot/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "datarobot" %}
+{% set version = "2.8.1" %}
+{% set sha256 = "338f5233a48251ea6aafae858f22e25e1610c55a20b17aafd92ef9da24cd5607" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - pytest-runner
```

Comment:
Is this a build requirement?

Reply:
Yes, unfortunately the package specifies `pytest-runner` in the setup.py `setup_requires` section. We can work to remove this dependency in the future but for this version of the package, it is required for building.

Reply:
I would have thought that `--no-deps` would have ignored `setup_requires` but oh well.
I guess it could be patched out, but maybe it isn't worth it at that point.

Reply:
The dep isn't getting downloaded by `pip` but instead `setuptools` has the ability to have its own dependency list but this functionality is correctly disabled in the conda build infrastructure it seems. Earlier builds were failing due to this: https://circleci.com/gh/conda-forge/staged-recipes/25415

I could go the patching route as you suggest. I'm not familiar enough with the conda infra to know what the trade offs are of listing the package as a build dep or patching the package's setup.py. Listing the extra build-dep seems pretty straightforward.

Reply:
I agree, at this point I wouldn't worry about it (especially since it is in the build deps and not the run deps). Generally the less unused build deps the faster the CI but here it shouldn't make enough difference for the aggravation to be worth it.